ctms/schemas/__init__.py

```diff
@@ -26,6 +26,7 @@
 from .mofo import MozillaFoundationInSchema, MozillaFoundationSchema
 from .newsletter import NewsletterInSchema, NewsletterSchema, UpdatedNewsletterInSchema
 from .product import ProductBaseSchema
+from .response import GDPRDeleteResponse
```

Comment:
We have responses schemas in `schemas/contact.py` (eg. `CTMSSingleResponse`). We could move it to your new module, or move `DeleteResponse` there